lib/src/widgets/video.dart

```diff
@@ -21,7 +21,7 @@ class VideoFrame {
   final int videoHeight;
   final Uint8List byteArray;
 
-  VideoFrame({
+  const VideoFrame({
```

Comment:
I think you should have separately a refactor pull request. Mix refactors and fullscreen feature makes difficult to track the codes